_data/simple-icons.json

```diff
@@ -8261,7 +8266,7 @@
         {
             "title": "ŠKODA",
             "hex": "4BA82E",
-            "source": "https://en.wikipedia.org/wiki/File:Skoda_Auto_logo_(2011).svg"
+            "source": "https://seeklogo.com/vector-logo/177065/skoda"
```

Comment:
As you mentioned, seeklogo is far from being an official source... We might use this one maybe:
https://commons.wikimedia.org/wiki/File:%C5%A0koda_Works_logo.svg